File: 01-pattern-problems/java/Pattern1.java
import java.util.Scanner;
/** 
 * @author Sourabh Ninawe
 */
class Pattern1{
    public void printPattern1(int n){
        for(int i=0;i<n;i++){
            for(int j=0;j<n;j++){
                System.out.print("*");
            }
            System.out.println();
        }
    }
    public static void main(String[] args) {
        while(true){
            System.out.println("Enter the number of lines to print pattern 1 in integer and -1 to exit");
            Scanner sc = new Scanner(System.in);
            int n = sc.nextInt();
            if(n<=0)
                break;
            else{
                Pattern1 p1 = new Pattern1();
                p1.printPattern1(n);
            }
        }
        
        
    }
}

File: 01-pattern-problems/python/pattern1.py

# Author: Sourabh Ninawe
class Pattern1:
    def print_pattern1(self, n):
        for i in range(n):
            for j in range(n):
                print('*', end='')
            print()
    
if __name__ == "__main__":
    while True:
        try:
            n = int(input("Enter the number of lines to print pattern 1 (integer, 0 or negative to exit): "))
        except ValueError:
            print("Please enter a valid integer.")
            continue
        if n <= 0:
            break
        else:
            p1 = Pattern1()
            p1.print_pattern1(n)
